sawmill-core/src/test/java/io/logz/sawmill/PipelineTest.java

```diff
@@ -185,21 +184,11 @@ public void shouldFailPipelineCreationOnNoProcessorFactoryConfigurationGiven() t
     }
 
     @Test(expected = SawmillException.class)
-    public void shouldFailProcessorCreationOnNoGeoIpConfigurationGiven() throws Exception {
-        new Pipeline.Factory().create("test", new PipelineDefinition(
-                singletonList(new ProcessorExecutionStepDefinition(new ProcessorDefinition(
-                        "geoIp", new HashMap<>()), null, null, null
-                )),
-                true
-        ));
-    }
-
-    @Test(expected = FactoryInstantiationException.class)
     public void shouldFailPipelineCreationOnGeoIpConfigurationInvalidFileGiven() throws Exception {
         new Pipeline.Factory(new GeoIpConfiguration("LICENSE"));
     }
 
-    @Test(expected = FactoryInstantiationException.class)
+    @Test(expected = SawmillException.class)
     public void shouldFailPipelineCreationOnGeoIpConfigurationFileDoesNotExist() throws Exception {
         new Pipeline.Factory(new GeoIpConfiguration("non-existing-file"));
     }
```

Comment:
`TestProcessorWithMandatoryDependencies` is unused, let's delete

Reply:
its deleted already